compiler/src/main/scala/dx/compiler/ApplicationCompiler.scala

```diff
@@ -235,34 +235,38 @@ case class ApplicationCompiler(typeAliases: Map[String, Type],
       case _ => Map.empty
     }
     // Add platform Docker image dependency to bundledDepends
-    val bundledDependsDocker: Option[JsValue] = applet.container match {
-      case DxFileDockerImage(_, dxfile) => {
-        val id = dxfile.id
+    val bundledDependsDocker: Option[JsValue] = if (applet.kind == ExecutableKindApplet) {
```

Comment:
Since both frag applet and the task applet have the container info, exclude the frag applet when making `bundledDepends`
@commandlinegirl @Gvaihir

Reply:
Not sure I 100% follow, where the frag applet will get its `bundledDepends` info from?

Reply:
https://github.com/dnanexus/dxCompiler/blob/3a8f4cd17f94356d8b4960317ecfea158fefd45f/compiler/src/main/scala/dx/translator/cwl/ProcessTranslator.scala#L578-L596

Here `availableDependencies` already contains the embedded applets (we translate them first then make the wrapper frag applet when translating the wf)

So here we use the container info fetched from the inner applet to make the new frag applet.

Reply:
Does this create any inefficiencies? Why do we want to create additional dependencies between the components?